experiments/ClimaEarth/components/ocean/prescr_seaice.jl

```diff
@@ -174,12 +172,11 @@ function ice_rhs!(du, u, p, _)
 
     F_conductive = @. params.k_ice / (params.h) * (params.T_base - Y.T_sfc) # fluxes are defined to be positive when upward
     rhs = @. (-F_turb_energy - F_radiative + F_conductive) / (params.h * params.ρ * params.c)
-
-    # do not count tendencies that lead to temperatures above freezing, and mask out no-ice areas
+    # If tendencies lead to temperature above freezing, set temperature to freezing
+    @. rhs = min(rhs, (T_freeze - Y.T_sfc) / p.dt)
+    # mask out no-ice areas
     area_mask = Regridder.binary_mask.(area_fraction)
-    threshold = zero(FT)
-    unphysical = @. Regridder.binary_mask.(T_freeze - (Y.T_sfc + FT(rhs) * FT(p.dt)), threshold) .* area_mask
-    parent(dY.T_sfc) .= parent(rhs .* unphysical)
+    parent(dY.T_sfc) .= parent(rhs .* area_mask)
```

Comment:
Would this work?

```suggestion
    dY.T_sfc .= rhs .* area_mask
```

Reply:
I think there might be a reason why we have parent before, but let me try.

Reply:
it works:)

Reply:
I suspected so